nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -176,6 +176,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `tilp2` was removed together with its module
 
+- `bird1` and its module has `services.bird` as well as `services.bird6` has been removed. Upgrade to `services.bird2`.
+
```

Comment:
We should probably also add the fact bird is running as root again, so it can drop privileges, instead of running with the scary capabilities.

Reply:
This was already the case before in fact.

Reply:
see https://github.com/NixOS/nixpkgs/pull/161193/files#diff-7b8f5c81bf510eb2bb574f1374914deebdafec8f5c001f9ea9983312438fa1e7L78